features/topology-map/plugins/org.opennms.features.topology.plugins.topo.linkd/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -35,46 +29,61 @@ xsi:schemaLocation="http://www.osgi.org/xmlns/blueprint/v1.0.0
         init-method="start"
         destroy-method="stop" />
 
-    <bean id="topologyProvider" scope="singleton" class="org.opennms.features.topology.plugins.topo.linkd.internal.LinkdTopologyProvider">
-        <argument ref="metricRegistry" />
-        <property name="onmsTopologyDao" ref="onmsTopologyDao"/>
-        <property name="topologyProviderInfo" ref="enlinkdTopologyProviderInfo"/>
+    <!-- EnhancedLinkd Default Topology Provider-->
+    <bean id="linkdTopologyProviderInfo" class="org.opennms.features.topology.api.topo.DefaultTopologyProviderInfo">
+        <property name="name" value="All"/>
+        <property name="description" value="This Topology Provider displays the topology information discovered by the Enhanced Linkd daemon. It uses the SNMP information of several protocols like OSPF, ISIS, LLDP and CDP to generate an overall topology."/>
+        <property name="supportsCategorySearch" value="true"/>
     </bean>
 
-    <bean id="topologyHopProvider" class="org.opennms.features.topology.api.support.VertexHopGraphProvider">
-        <argument ref="topologyProvider"/>
+    <bean id="linkdTopologyProvider" scope="singleton" class="org.opennms.features.topology.plugins.topo.linkd.internal.LinkdTopologyProvider">
+        <argument index="0" value="enlinkd" />
+        <argument index="1" ref="metricRegistry" />
+        <argument index="2"> 
```

Comment:
Let's get rid of the `index=?` calls, they are unnecessary as the index is implicit by the position.